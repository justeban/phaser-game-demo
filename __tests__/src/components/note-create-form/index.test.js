import React from 'react';
import renderer from 'react-test-renderer';

import NoteForm from '../../../../src/components/note-create-form/index.js';

describe('<Dashboad /> (Enzyme Test)', () => {

  it('testing intial state of NoteForm at start', () => {
    let app = shallow(<NoteForm />);
    expect(app.find('form').exists()).toBeTruthy();
  });

});

describe('<NoteForm /> (Snapshot Test)', () => {

  it('renders cleanly', () => {
    const component = renderer.create(
      <NoteForm />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
