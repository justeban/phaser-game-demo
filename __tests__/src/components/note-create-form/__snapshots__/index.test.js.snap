// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoteForm /> (Snapshot Test) renders cleanly 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    name="title"
    type="text"
  />
  <input
    name="content"
    type="text"
  />
  <input
    type="submit"
  />
</form>
`;
